service/src/main/java/uk/gov/hmcts/reform/fpl/config/GeneralInboxLookupConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.fpl.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@lombok.Data
+@Service
+public class GeneralInboxLookupConfiguration {
```

Comment:
why _General_, what exactly are we looking to model? Also, if annotating with `@Service` why is the class `Configuration`?

Reply:
Yep really not sold on the naming here. Could this not be a Lombok model?

Reply:
changed it to `PublicLawEmailLookupConfiguration`. Its Configuration. Removed `@services` tag and added `@configuration`.